File: chapter1(1-50)/sixth/sum_difference.cpp
#include<iostream>
using namespace std;
int main(){
   int i;
  long long int sum=0;
  int sum_1=0;
  long long  int sum_2=0;
  long long int sum_3;
  for(i=1;i<=100;i++){
      sum=sum+i*i;
      sum_1=sum_1+i;
  }
  sum_2=sum_2+sum_1*sum_1;
  sum_3=sum_2-sum;
  cout<<sum_3<<endl;
  return 0;
  
  
}

File: chapter1(1-50)/sixth/sum_difference.rs
fn sum_difference(n:u64)->u64{
            let mut sum:u64=0;
            let mut sum_1:u64=0;
            let mut sum_2:u64=0;
            let  sum_3:u64;
            for i in 1..=n{
                sum=sum+i*i;
                sum_1=sum_1+i;
            }
            sum_2=sum_2+sum_1*sum_1;
            sum_3=sum_2-sum;
            sum_3
        }
        fn main(){
            let  result:u64;
            result=sum_difference(100);
            println!("{}",result);

        }
